apis/sesv2/v1alpha1/zz_email_i_dentity.go

```diff
@@ -0,0 +1,113 @@
+/*
```

Comment:
I think this should be `identity`, not `i_dentity` in `zz_email_i_dentity.go`. I wonder if this has been fixed upstream in the latest version of https://github.com/aws-controllers-k8s/code-generator/ - maybe we should bump that as a PR first, or create an issue there.

Reply:
This has been fixed upstream in code-generator

Reply:
Does it mean I have to use different commit for `CODE_GENERATOR` to get this item fixed?

Reply:
yes, and there's a PR for that already: https://github.com/crossplane/provider-aws/pull/801